src/Bicep.Core.IntegrationTests/AssertTests.cs

```diff
@@ -21,13 +20,41 @@ public class AssertTests
         private ServiceBuilder ServicesWithAsserts => new ServiceBuilder()
             .WithFeatureOverrides(new(TestContext, AssertsEnabled: true));
 
-
         [TestMethod]
         public void Asserts_are_disabled_unless_feature_is_enabled()
         {
             var services = new ServiceBuilder().WithFeatureOverrides(new());
 
-            var result = CompilationHelper.Compile(services, @"
+            string fileContent = @"
```

Comment:
We can change the verbatim string to [raw string literals](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-11.0/raw-string-literal).

Reply:
Is there a reason we may prefer one over the other?

Reply:
Verbatim strings keep any whitespace within them. If we had indented the Bicep code to be aligned with the C# code, the resulting Bicep code would include all of the corresponding indentation whitespace. Raw string literals remove the extra indentation. Another befinit of raw string literals is that it eliminates the need for escaping `"`, though for Bicep code there is no difference since Bicep strings are quoted with `'`.

Reply:
ah I see. The difference in handling the whitespace makes sense.